src/components/MoviesList/MoviesList.tsx

```diff
@@ -2,17 +2,17 @@ import React from 'react';
 
 import './MoviesList.scss';
 import { MovieCard } from '../MovieCard';
-import { Movie } from '../../types/Movie';
+import { MovieData } from '../../types/MovieData';
 
 type Props = {
-  movies: Movie[];
+  movies: (MovieData | null)[];
```

Comment:
Same here. You don't need to render the list if you don't have movies at all
```suggestion
  movies: MovieData[];
```